apps/institution-service/cypress/e2e/institutions.cy.ts

```diff
@@ -496,6 +496,63 @@ describe("/institutions", () => {
     );
   });
 
+  it("gets sorted institution list with default sort options", () => {
+    cy.request({
+      url: `http://localhost:${PORT}/institutions?aggregatorName=testExampleA`,
+      method: "GET",
+      headers: {
+        Authorization: createAuthorizationHeader(SUPER_USER_ACCESS_TOKEN_ENV),
+      },
+    }).then((response: Cypress.Response<{ body: unknown }>) => {
+      const institutionResponse =
+        response.body as unknown as PaginatedInstitutionsResponse;
+
+      expect(response.status).to.eq(200);
+      expect(institutionResponse.totalRecords).to.eq(3);
+      expect(institutionResponse.institutions[0].id).to.eq(
+        "3b561893-e969-4a2c-9e58-3b81b203cdc1",
+      );
+    });
+  });
+
+  it("gets sorted institution list with a single, custom sort option", () => {
+    cy.request({
+      url: `http://localhost:${PORT}/institutions?aggregatorName=testExampleA&sortBy=id:desc`,
+      method: "GET",
+      headers: {
+        Authorization: createAuthorizationHeader(SUPER_USER_ACCESS_TOKEN_ENV),
+      },
+    }).then((response: Cypress.Response<{ body: unknown }>) => {
+      const institutionResponse =
+        response.body as unknown as PaginatedInstitutionsResponse;
+
+      expect(response.status).to.eq(200);
+      expect(institutionResponse.totalRecords).to.eq(3);
+      expect(institutionResponse.institutions[0].id).to.eq(
+        "7a909e62-98b6-4a34-8725-b2a6a63e830a",
+      );
+    });
+  });
+
+  it("gets sorted institution list with multiple, custom sort options", () => {
```

Comment:
Why do we need multiple if there are only UI controls for 1?

Reply:
Good point about the UI. The default, hard-coded sort was for two columns, so I thought we needed to support that use-case.

Reply:
You can see that in the original code: 
<img width="371" alt="image" src="https://github.com/user-attachments/assets/64c1a3dc-625a-4392-96af-082df75f2ccd" />

Reply:
I think that solution sounds good @wesrisenmay-mx. The reason we added the secondary sort was because the createdAt field was exactly the same for most institutions and it was messing up the pagination.

I agree with keeping the name as secondary sort for all other sorts. If they are sorting by name then you probably don't need a secondary sort, but making it sort by `id` seems like it wouldn't hurt anything.